advocacy_docs/edb-postgres-ai/ai-accelerator/retrievers/concepts.mdx

```diff
@@ -1,66 +1,66 @@
 ---
-title: "Retrievers - Concepts"
+title: "Retrievers concepts"
 navTitle: "Concepts"
 deepToC: true
 description: "The concepts behind using retrievers in AI Accelerator Pipelines."
 ---
 
-Retrievers can be used for semantic search or for RAG (retrieval augmented generation).
-A configured retriever can automatically compute embeddings (vectors) for our input data.
-Once this is done, the retriever offers a number of interfaces to return data based on a query.
+Retrievers can be used for semantic search or for retrieval augmented generation (RAG).
+A configured retriever can compute embeddings (vectors) for your input data.
+After the computing is done, the retriever offers interfaces to return data based on a query.
```

Comment:
Why is there a level 2 heading called Concepts in a topic called Concepts? Should everything move up a level? If not, some text between the level2 and first level 3 head would be nice. And if so, I think the opening paragraph needs to maybe mention concepts? Like "These concepts will help you to understand how to approach configuring your retrievers." (Totally made up and I'm sure not accurate, but along those lines.